lib/BarChart.js

```diff
@@ -20,5 +20,8 @@ const iface = {
 };
 
 export default requireNativeComponent('RNBarChart', iface, {
-  nativeOnly: { onSelect: true }
+  nativeOnly: {
+    onSelect: true,
+    onMessage: true
```

Comment:
nit, should we call these something more specific like onGestureEvent?

Reply:
RN actually has specific names of events it maps (e.g. java `topChange` --> js `onChange`), so used `onMessage` since there are events beyond what's the supported/generic ones.

Resources:
[mapping in source code](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/uimanager/UIManagerModuleConstants.java)
[react-native docs](https://facebook.github.io/react-native/docs/native-components-android.html#events)